CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 0.2.2 - 2019-10-24
+
+## Added
+
+ - Added `deep_with_changes` helper method on CSTNode.
+ - Added metadata support to matchers.
```

Comment:
Maybe we can mention `MatchMetadata` to make user easier to find this new feature?